ceph_iscsi_config/lun.py

```diff
@@ -241,7 +241,6 @@ def __init__(self, logger, pool, image, size, allocating_host):
         # only uses shortname so it needs to be converted to shortname format
         self.allocating_host = allocating_host.split('.')[0]
 
-        self.owner = ''     # gateway that owns the preferred path for this LUN
```

Comment:
In ```gwcli```, running "/disks info <disk_id>" on an unassigned disk will result in a failure: ```AttributeError: 'Disk' object has no attribute 'owner'```

See https://github.com/ceph/ceph-iscsi/blob/master/gwcli/storage.py#L586